src/setup/node.rs

```diff
@@ -301,6 +301,7 @@ mod test {
     const SLEEP: Duration = Duration::from_millis(100);
 
     #[tokio::test]
+    #[ignore = "use only when changing src/setup files"]
```

Comment:
I haven't tried it before, but perhaps this `ignore` attrib could be made module-level and apply to all these tests at once :thinking:?